template.yaml

```diff
@@ -210,6 +210,189 @@ Resources:
       Tags:
         - Key: "AppComponent"
           Value: "trapheus:iamrole"
+  
+  RestoreDBClusterFromSharedSnapshotSSMDocument:
+    Type: 'AWS::SSM::Document'
+    Properties:
+      Name: Trapheus-RestoreDatabaseClusterFromSharedSnapshot
+      DocumentFormat: YAML
+      DocumentType: Automation
+      Content:
+        schemaVersion: '0.3'
+        assumeRole: '{{ AutomationAssumeRole }}'
```

Comment:
Will this be a cross account tole https://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html

Reply:
yes - it needs to be additionally configured in the target account - will raise a separate PR with the required CFT

Reply:
ok